heart_animation_2/lib/extensions.dart

```diff
@@ -0,0 +1,6 @@
+import 'dart:developer' as devtools show log;
+
+extension Log on Object {
+  /// just for debugging purposes
+  void log() => devtools.log(toString());
+}
```

Comment:
Not related to this file, let's delete [home.dart](https://github.com/hiashutoshsingh/flutter_heart_challenge/pull/7/files#diff-11a1406e0b19a26aace7ac992689a58a22aa482f308eb4aae525832230f00650) file as there is no use of it.

Reply:
deleted home.dart file